src/main/java/com/nirima/jenkins/plugins/docker/DockerComputerLauncher.java

```diff
@@ -57,8 +56,6 @@ private static SSHLauncher getSSHLauncher(InspectContainerResponse detail, Docke
             String host = hostUrl.getHost();
 
             LOGGER.log(Level.INFO, "Creating slave SSH launcher for " + host + ":" + port);
-            
-            PortUtils.waitForPort(host, port);
```

Comment:
I'm not clear why this is any better than what's already there : 
PortUtils.waitForPort is already attempting to wait for the remote SSH to be available before continuing ?